apps/e2e/action-bars-storybook/src/app/summary-action-bar/summary-action-bar.component.ts

```diff
@@ -1,15 +1,30 @@
-import { Component, Input, OnDestroy, inject } from '@angular/core';
+import { Component, Injectable, Input, OnDestroy, inject } from '@angular/core';
+import { SkySummaryActionBarError } from '@skyux/action-bars';
 import { SkyModalInstance, SkyModalService } from '@skyux/modals';
 
 import { SummaryActionBarModalComponent } from './summary-action-bar-modal.component';
 
+@Injectable()
+export class SABModalContext {
+  public errors: SkySummaryActionBarError[] | undefined;
+}
+
 @Component({
   selector: 'app-summary-action-bar',
   templateUrl: './summary-action-bar.component.html',
   styleUrls: ['./summary-action-bar.component.scss'],
   standalone: false,
 })
 export class SummaryActionBarComponent implements OnDestroy {
+  @Input()
+  public set errors(errorsValue: SkySummaryActionBarError[]) {
```

Comment:
Minor - but is there a reason that we aren't just doing a normal input here typed `SkySummaryActionBarError[] | undefined`?